streams/src/test/java/org/apache/kafka/streams/integration/AbstractResetIntegrationTest.java

```diff
@@ -157,12 +156,16 @@ private void prepareConfigs() {
     private static final long CLEANUP_CONSUMER_TIMEOUT = 2000L;
     private static final int TIMEOUT_MULTIPLIER = 5;
 
-    private final TestCondition consumerGroupInactiveCondition = new TestCondition() {
+    private class ConsumerGroupInactiveCondition implements TestCondition {
         @Override
         public boolean conditionMet() {
-            return adminClient.describeConsumerGroup(testId + "-result-consumer", 0).consumers().get().isEmpty();
+            try {
+                return adminClient.describeConsumerGroup(appID, 0).consumers().get().isEmpty();
+            } catch (final TimeoutException e) {
```

Comment:
This fix seems unrelated to the null-delete semantics thing -- I think we should split it into two PRs.

Reply:
Yes, this is not intentional; do not know how it gets in, will remove.